test/e2e/testing-manifests/storage-csi/external-snapshotter/volume-group-snapshots/run_group_snapshot_e2e.sh

```diff
@@ -28,8 +28,6 @@ set -o errexit -o nounset -o xtrace
 # parallel testing is enabled. Using LABEL_FILTER instead of combining SKIP and
 # FOCUS is recommended (more expressive, easier to read than regexp).
 #
-# GA_ONLY: true  - limit to GA APIs/features as much as possible
```

Comment:
https://cs.k8s.io/?q=run_group_snapshot_e2e&i=nope&literal=nope&files=&excludeFiles=&repos=

most of this script was inherited from hack/ci/e2e-k8s.sh in the kind repo, and the jobs using this script do not use features like GA_ONLY or logging format

Reply:
I'm working on porting these changes to the other scripts in kind and test-infra

https://github.com/kubernetes-sigs/kind/pull/4046